core/build.gradle.kts

```diff
@@ -1,56 +1,42 @@
-
-
+import buildsrc.config.createWardenPom
+import buildsrc.config.defaults
+import buildsrc.config.publish
 
 plugins {
-    kotlin("multiplatform")
-    id("com.jfrog.artifactory")
-    id("maven-publish")
-}
-
-repositories {
-    mavenCentral()
-    jcenter()
+    buildsrc.convention.`kotlin-multiplatform`
+    buildsrc.convention.`artifactory-publish`
 }
 
 val assertKVersion: String by project
 val mockkVersion: String by project
 val kotlinVersion: String by project
 
 kotlin {
-    jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = "11"
-        }
-    }
     sourceSets {
         val commonMain by getting {
             dependencies {
-                implementation(kotlin("stdlib-common"))
                 implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3")
             }
         }
         val commonTest by getting {
             dependencies {
-                implementation(kotlin("test-common"))
-                implementation(kotlin("test-annotations-common"))
-                implementation("io.mockk:mockk-common:$mockkVersion")
-                implementation("com.willowtreeapps.assertk:assertk-common:0.14")
+                implementation(kotlin("test"))
+                implementation("io.mockk:mockk:$mockkVersion")
+                implementation("com.willowtreeapps.assertk:assertk:$assertKVersion")
             }
         }
 
-        // Default source set for JVM-specific sources and dependencies:
-        jvm().compilations["main"].defaultSourceSet {
+        val jvmMain by getting {
+            // Default source set for JVM-specific sources and dependencies:
             dependencies {
-                implementation(kotlin("stdlib"))
-                implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
+                implementation(kotlin("reflect"))
             }
         }
-        // JVM-specific tests and their dependencies:
-        jvm().compilations["test"].defaultSourceSet {
+
+        val jvmTest by getting {
```

Comment:
Changing `jvm().compilations["test"].defaultSourceSet` is a unnecessary but I like it if it's consistent with `commonMain` :)